remove windows releases (#23)
datatrails/veracity: README.md

## Installation

> Veracity provides native binaries for [Mac OS](#mac-install) and [Linux](#linuxwsl-install) on the [releases](https://github.com/datatrails/veracity/releases) page.

Reviewer: ```suggestion
Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows (WSL)
```

> Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows (WSL) on the [releases](https://github.com/datatrails/veracity/releases) page.

Reviewer 2: I don't think we should say that
